components/match2/wikis/trackmania/brkts_wiki_specific.lua

```diff
@@ -0,0 +1,22 @@
+---
+-- @Liquipedia
+-- wiki=trackmania
+-- page=Module:Brkts/WikiSpecific
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local WikiSpecific = Table.copy(Lua.import('Module:Brkts/WikiSpecific/Base', {requireDevIfEnabled = true}))
+
+function WikiSpecific.getMatchGroupContainer(matchGroupType)
```

Comment:
do you have the need for score3 (or score2 in non reset matches)?
if not kick this function

Reply:
We now have with the upcoming Trackmania World Tour competition